packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
Is it possible to address the issue rather than ignoring it by including the variables inside the dependancy array?

Reply:
@matin-binary the issue is if I include the variable submission_status in the dependency array which is causing the warning then the component gets re renders every time there is a change in submission_status which is wrong behavior in the proof of identity process.